### PR TITLE
Yukebrillianth

### PR DESCRIPTION
**Pull request by yukebrillianth**
## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-id.md [hunk 1]
```diff
@@ -74,6 +76,11 @@
 * [Workshop Python 101](http://sakti.github.io/python101/)
 
 
+### Sistem Operasi
+
+* [Belajar Sistem Operasi](http://onnocenter.or.id/pustaka/)
```
Comment (eshellman) [inline free-programming-books-id.md:81]: not a book

File: free-programming-books-id.md [hunk 2]
```diff
@@ -62,6 +62,8 @@
 * [Pemrograman Web dengan PHP dan MySQL](http://achmatim.net/2009/04/15/buku-gratis-pemrograman-web-dengan-php-dan-mysql/)
 * [PHP: The Right Way Bahasa Indonesia](http://id.phptherightway.com/#site-header/)
 * [Tutorial Ebook PHP](http://www.ilmuwebsite.com/ebook-php-free-download)
+* [Pemrograman Berorientasi Objek Dengan PHP5](https://endangcahyapermana.files.wordpress.com/2016/03/belajar-singkat-pemrograman-berorientasi-objek-dengan-php5.pdf)
```
Comment (eshellman) [inline free-programming-books-id.md:65]: alphabetiziation

File: free-programming-books.md [hunk 3]
```diff
@@ -2306,6 +2306,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [The Standard Python Library](http://effbot.org/librarybook/) - Fredrik Lundh
 * [Think Complexity](http://greenteapress.com/complexity/) - Allen B. Downey (2nd Edition) (PDF, HTML)
 * [Tiny Python 3.6 Notebook](https://github.com/mattharrison/Tiny-Python-3.6-Notebook) - Matt Harrison (3.6)
+* [Python Notes for Professionals](https://books.goalkicker.com/PythonBook/)
```
Comment (eshellman) [inline free-programming-books.md:2309]: P < T

**Review by eshellman — COMMENTED** [review]
Check TravisCI utput for errors

File: free-programming-books-id.md (hunk 1 above)
Comment (yukebrillianth) [reply free-programming-books-id.md:81]: I do tomorrow

**Pull request closed without merging**